### PR TITLE
kalign 3.3.1

### PR DESCRIPTION
**Pull request by biomadeira**
- [ x] Have you followed the [guidelines for contributing](https://github.com/brewsci/homebrew-bio/blob/master/CONTRIBUTING.md)?
- [ x] Have you checked that there aren't other open [pull requests](https://github.com/brewsci/homebrew-bio/pulls) for the same formula update/change?
- [x ] Have you built your formula locally with `brew install --build-from-source FORMULA`, where `FORMULA` is the name of the formula you're submitting?
- [ x] Does your build pass `brew audit --strict FORMULA` (after doing `brew install FORMULA`)?

-----

### CODE REVIEW

File: Formula/kalign.rb [hunk 1]
```diff
@@ -2,8 +2,8 @@ class Kalign < Formula
   # cite Lassmann_2019: "https://doi.org/10.1093/bioinformatics/btz795"
   desc "SIMD accelerated multiple sequence alignment"
   homepage "https://github.com/TimoLassmann/kalign"
-  url "https://github.com/TimoLassmann/kalign/archive/v3.3.tar.gz"
-  sha256 "bb49b531ad320881154976194acb8cc0c519715b0d422eac7a171cc246f88d59"
+  url "hhttps://github.com/TimoLassmann/kalign/archive/v3.3.1.tar.gz"
```
Comment (sjackman) [inline Formula/kalign.rb:5]:
```suggestion
  url "https://github.com/TimoLassmann/kalign/archive/v3.3.1.tar.gz"
```
Fix the typo

**Review by sjackman — CHANGES_REQUESTED** [review]
Fix the typo.

**Comment by biomadeira** [comment]
@sjackman Sorry for the typo 😬

**Review by sjackman — APPROVED** [review]

**Comment by sjackman** [comment]
No worries. Thank you for your first contribution to Brewsci/bio, Fábio! 🥇

**Pull request merged**